packages/core/core/src/domain/content-type/index.ts

```diff
@@ -133,6 +133,7 @@ const pickSchema = (model: Schema.ContentType) => {
       'options',
       'pluginOptions',
       'attributes',
+      'kind',
```

Comment:
Why is this part of this PR? Was this missing?

Reply:
I was getting a type error I'll recheck

Reply:
Yeah I need that fix